docs/quickstart/introduction.md

```diff
@@ -0,0 +1,11 @@
+---
+sidebar_label: "Introduction"
+description: "Nym connect quickstart"
+hide_title: false
+---
+
+# Introduction
+
+Nym Connect is a one-button interface that is easy to use. With Nym connect, you get 100% privacy on applications you use everyday. Presently, Nym connect has support for telegram, keybase and electrum wallets. This means that you can run sock5 proxy on these applications with Nym connect. More services coming soon.
```

Comment:
remove the line "With Nym connect, you get 100% privacy on applications you use everyday." - we dont have SURBs yet and a few other things so this isn't strictly true

Reply:
Change 'Nym connect has support for telegram, keybase and electrum wallets. This means that you can run sock5 proxy on these applications with Nym connect. More services coming soon.' to something like 'Presently Nym Connect allows you to send traffic for supported applications through the mixnet, shielding your metadata from the wider internet. NymConnect currently offers support for Telegram, Keybase, and the Electrum Bitcoin wallet. More services will be added soon."